renovate.json

```diff
@@ -24,7 +24,7 @@
       ],
```

Comment:
How about removing the newline from matchstrings like this.
```
      "matchStrings": [
        "^(?<currentValue>[0-9.]+)"
      ],
```
then the renovate stuff maybe works independent of the linebreak.

Reply:
https://docs.renovatebot.com/configuration-options/#matchstringsstrategy

I think all three will match every possible combination without the newline if I understand correctly.

Reply:
It doesn't seem that way. I've ran this locally with the current mr it's
```
                 "replaceString": "1.22.4\n",
```
with the adjusted
```
                 "replaceString": "1.22.4",
```
So I guess currently it tries to replace with newline.

Reply:
How did you get it running locally ... I tried every combination I can think of I always get this error. The CLI interface documentation is just ...

npx renovate --token $GITHUB_TOKEN --dry-run 1 --platform github gotify/build # and variations

```
 "loggerErrors": [
         {
           "name": "renovate",
           "level": 50,
           "logContext": "jyimdCaCMKo1Obj9ShKxc",
           "repository": "1",
           "err": {
             "message": "platform-unknown-error",
             "stack": "Error: platform-unknown-error\n    at Proxy.initRepo (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/modules/platform/github/index.ts:512:13)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at getPlatformConfig (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/repository/init/apis.ts:97:26)\n    at initApis (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/repository/init/apis.ts:109:12)\n    at initRepo (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/repository/init/index.ts:53:12)\n    at Object.renovateRepository (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/repository/index.ts:62:14)\n    at attributes.repository (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/global/index.ts:218:11)\n    at start (/home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/workers/global/index.ts:203:7)\n    at /home/yume/.local/share/nvm/v22.9.0/lib/node_modules/renovate/lib/renovate.ts:18:22"
           },
           "msg": "Repository has unknown error"
         }
       ]
```

Reply:
```
docker run -it -e LOG_LEVEL=debug -e RENOVATE_TOKEN=GITHUB_PERSONAL_TOKEN -v $PWD:/usr/src/app --rm renovate/renovate renovate --platform=local
```
I've used this, and $PWD is a locally checked out gotify/build repo.